src/api/user/controller.js

```diff
@@ -1,8 +1,12 @@
 /* eslint-disable no-unused-vars */
 
 import { sendJson } from 'utils/api';
-import { serializeUser } from './model';
+import User, { serializeUser } from './model';
 
-export const getAll = (req, res, next) => res.status(200).json({ message: 'users getAll api' });
+export const getAll = async (req, res, next) =>
+  User.find({})
```

Comment:
please add `serializeUser` for each user in result, check `getAllArticles` for example